.github/workflows/npm-publish.yml

```diff
@@ -28,8 +25,12 @@ jobs:
         run: npm run build
 
       - name: Define NPM tag
-        run: if [[ ${VERSION} == *"alpha"* || ${VERSION} == *"beta"* || ${VERSION} == *"rc"* ]]; then echo "NPM_TAG=next"; else echo "NPM_TAG=latest"; fi >> $GITHUB_ENV
-      - run: echo ${NPM_TAG}
+        run:
+          if [[ ${VERSION} == *"alpha"* || ${VERSION} == *"beta"* || ${VERSION} == *"rc"* ]]; then
```

Comment:
is this change required? Isnt it this change going to cause issue due to use of `echo ${NPM_TAG}`  immediately after setting it via $GITHUB_ENV, because echo "NPM_TAG=..." >> $GITHUB_ENV sets the variable for future steps, not in the current one.

Reply:
I reverted the changes.